atomsci/ddm/utils/datastore_functions.py

```diff
@@ -1741,7 +1748,7 @@ def search_files_interactive (bucket='all', client=None, to_return='df', display
         value = value.split(",")
         value = [x.strip(' ') for x in value]
         print('currently value=', value)  #delete?
-        if type(values_for_key) == np.ndarray:
+        if type(values_for_key) is np.ndarray:
```

Comment:
test?

Reply:
This will need to wait for a later release when the datastore is back up.